.github/workflows/fmt-all.yml

```
@@ -35,7 +35,7 @@ jobs:
             full-matrix: true
             run-with-macos: ${{ inputs.run-with-macos }}
             use-self-hosted-linux: ${{ inputs.use-self-hosted-linux }}
-        secrets: inherit
+        secrets: {}
```

actionlint (1.7.12)

[error] "secrets" section should not be empty. please remove this section if it's unnecessary

(syntax-check)